src/hooks/useCategories.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const fetchCategories = () => {
+  return axios
+    .get("https://nc-games-portfolio.herokuapp.com/api/categories")
+    .then(({ data: categories }) => {
+      return categories;
+    });
+};
+
+export const useCategories = () => {
+  const [categories, setCategories] = useState("all");
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchCategories()
+      .then(({ categories }) => {
+        setIsLoading(true);
+        setCategories(categories);
+        setIsLoading(false);
+      })
+      .catch((err) => setError(err));
+  });
```

Comment:
No dependency array - this useEffect is always running!

Reply:
Thanks! I found this when I started looking at the next ticket, I just forgot the fix wasn't on this branch.